README.md

```diff
@@ -1,9 +1,26 @@
 # cloudfixLinter-demo-cloudformation
 
-For Demo Testing 
- 1. Install the extension
- 2. Make sure that a stack is already deployed to your AWS account
- 3. AWS account credentials are required in the default profile in `~/.aws/credentials`
+## Using Extension with Cloudformation
+ 1. Login to AWS using terminal by any of the AWS login methods:
+    1. `aws configure` to setup your AWS account (in default profile)
```

Comment:
these feel like steps, make it obvious that they are options

Reply:
Done